### DISCUSSION TITLE
PnP Search web part - Enable Stemming

### DISCUSSION THREAD

**Discussion by samrutur** [microsoft-search/pnp-modern-search#4769] [Ideas]
Is there a way to Enable Stemming in the search results web part? This is a supported SharePoint feature in the search API. https://learn.microsoft.com/en-us/sharepoint/dev/general-development/sharepoint-search-rest-api-overview#enablestemming

**Comment by wobba** [comment]
Sounds like a good feature to enable for sure :) Weird that we never have.

**Comment by samrutur** [comment]
Thanks. That would be a good feature to have.

**Comment by MartinLaplante** [comment]
As far as I understand the linguistic search features, and after some testing, I believe this stemming is on by default, and the stemming language rules are determined by the DetectedLanguage of the document.

So the question is can it be turned off for individual queries, as the API also allows?  Again, according to the documentation this should be feasible for most agglutinative and fusional languages (which includes most Indo-European languages) except Arabic, Estonian, Finnish, Hebrew, Hungarian, Korean, Latvian, Lithuanian, Slovak, and Turkish.  Those languages use reductive stemming only (running → run) but expansion stemming also goes in the other direction.
